### ISSUE TITLE
[2.12.1 Regression] 'LinterStats' object has no attribute 'get'

### ISSUE THREAD

**Issue by dzhengfy** [PyCQA/pylint#5426]
### Bug description

Refer link: https://github.com/autotest/tp-libvirt/runs/4330039462?check_suite_focus=true
Tp-libvirt project is using pylint for python checking. With pylint 2.11.0, everything is PASS. After upgrading to pylint 2.12, every PR checking is failed with below error message.

Run inspekt checkall --disable-style E501,E265,W601,W605,E402,E722,E741 --no-license-check
PEP8 disabled: E501,E265,W601,W605,E402,E722,E741
Pylint disabled: W,R,C,E1002,E1101,E1103,E1120,F0401,I0011
Pylint enabled : W0611
License check: disabled
'LinterStats' object has no attribute 'get'
Error: Process completed with exit code 1.

### Command used

```shell
inspekt checkall --disable-style E501,E265,W601,W605,E402,E722,E741 --no-license-check
```

### Pylint output

```shell
'LinterStats' object has no attribute 'get'
```

### Expected behavior

Checking is PASSed.

### Pylint version

```shell
pylint-2.12.1
```

**Comment by DanielNoord** [comment]
This is not actually a bug in `pylint`, but a bug in `inspektor` which is using elements of `pylint`'s API that we did not consider public.
A PR has already been opened with `inspektor` to fix this, see https://github.com/avocado-framework/inspektor/pull/41.

I'll respond to that PR with a better fix to solve the issue!

**Comment by dzhengfy** [comment]
@DanielNoord Thank you so much for your help.

**Comment by Pierre-Sassoulas** [comment]
The MR in avocado-framework/inspektor#41 seems well on it's way with the precious help from @DanielNoord I added this issue to 2.13, so we close it at some point.

**Comment by DanielNoord** [comment]
Closing as the downstream issue has been fixed! Thanks for the report @dzhengfy!
